### PR TITLE
Simple_adding Coderbyte Challenge

### PR DESCRIPTION
**Pull request by ghost**
Added code to create a simple adding program.

### CODE REVIEW

File: examples/simple_adding.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def simple_adding(number)
+  i = 0
+  sum = 0
+  number.times do
```
Comment (omidbachari) [inline examples/simple_adding.rb]: `.times` comes with a built in counter 😃. Name it whatever you want. I've named it `counter`. To make it an easy refactor, just use the name `i`. 

``` ruby
number.times do |counter|
  # your code
end
```

Comment (omidbachari) [reply examples/simple_adding.rb]: BTW - You will no longer have to increment it, if you use the built in counter. `times` will do that for you, as it goes through each iteration.

File: examples/simple_adding.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def simple_adding(number)
+  sum = 0
+  number.times do |i|
+    i = i + 1
```
Comment (omidbachari) [inline examples/simple_adding.rb]: the `times` method automatically increments your local variable named `i` as the loop iterates. no need to increment the variable yourself.

Comment (ghost) [reply examples/simple_adding.rb]: Ok, so I can remove line 4?

Comment (omidbachari) [reply examples/simple_adding.rb]: Try it

Comment (ghost) [reply examples/simple_adding.rb]: That did not work.

**Pull request merged**